edith-step/src/App.js

```diff
@@ -1,15 +1,31 @@
 import React, {Component} from 'react';
-import ReceiptInput from './ReceiptInput';
-// import "./App.css";
 
+import ReceiptInput from './components/ReceiptInput';
+import TopNavbar from './components/TopNavbar';
 
-/** Main webpage for the website. */
+import './App.css';
+
+/**
+ * Main webpage for the website.
+ */
 class App extends Component {
-  /** Renders the main webpage. */
+
```

Comment:
Lint error: "Block must not be padded by blank lines"

Please fix.

Reply:
Changes made.